meteor/client/ui/SegmentTimeline/SegmentContextMenu.tsx

```diff
@@ -36,8 +36,8 @@ export const SegmentContextMenu = withTranslation()(
 			const timecode = this.getTimePosition()
 			const startsAt = this.getPartStartsAt()
 
-			const isCurrentPart =
-				(part && this.props.playlist && part.instance._id === this.props.playlist.currentPartInstanceId) || undefined
+			const isCurrentPart = false
+			// (part && this.props.playlist && part.instance._id === this.props.playlist.currentPartInstanceId) || undefined
```

Comment:
This line should be reverted before merging, as non-ab setups havent been resolved yet